sourceview/markdown.lang

```diff
@@ -319,7 +319,21 @@
       <start>^```$</start>
       <end>^```$</end>
     </context>
+   
+    <!-- Examples:
+         Here's a literal HTML tag: ``<p>``.
+         ``The grave accent (`) is used in Markdown to indicate a code span.``
+         ``Here's another code span containing enter and four space before every lines of coded content
+    -->
+    <context id="2-backticks-code-span" class="no-spell-check mono" style-ref="code">
+      <match>(?&lt;!`)``[^`]+((`|`{3,})[^`]+)*``(?!`)</match>
+    </context>
 
+    <context id="3-backticks-code-span" class="no-spell-check mono" style-ref="code">
```

Comment:
four-space codeblocks have been removed, due to them interfering with enumerate/itemize lists.

See https://github.com/blackhole89/notekit/issues/32 & https://github.com/blackhole89/notekit/pull/77 for more info.

Reply:
it's not wise to rm it, it contains two enter and four space, I can't say it's conflict